databricks/koalas/series.py

```diff
@@ -1452,6 +1452,109 @@ def clip(self, lower: Union[float, int] = None, upper: Union[float, int] = None)
         """
         return _col(self.to_dataframe().clip(lower, upper))
 
+    def drop(self, labels=None, axis=0, index: Union[str, List[str]] = None, level=None):
+        """
+        Return Series with specified index labels removed.
+
+        Remove elements of a Series based on specifying the index labels.
+        When using a multi-index, labels on different levels can be removed by specifying the level.
+
+        Parameters
+        ----------
+        labels : single label or list-like
+            Index labels to drop.
+        axis : 0, default 0
+            Redundant for application on Series.
```

Comment:
Do we need this for `Series`? I'm not sure how it works for `Series`.

Reply:
@ueshin , Thanks for comment! actually since i don't have a complete understanding of Series yet, just mimic of pandas one. i'll take a look at this again, and change this part properly.